docs/userGuide/syntax/boxes.mbdf

```diff
@@ -101,6 +104,9 @@
 <box type="info" icon=":rocket:">
     info, with rocket icon
 </box>
+<box type="definition" light>
+    light definition
+</box>
 </span>
 </include>
```

Comment:
We may want to show the full set of box examples once more in the light mode i.e., one set in default styles and one set in light styles